org.jacoco.core.test/src-java8/org/jacoco/core/test/validation/BadCycleInterfaceTest.java

```diff
@@ -26,12 +26,15 @@ public BadCycleInterfaceTest() throws Exception {
 
 	@Test
 	public void test() throws Exception {
-		if (!System.getProperty("java.version").startsWith("9-ea")) {
+		if (System.getProperty("java.version").startsWith("9-ea")) {
 			// JDK-9042842
+			assertLine("baseclinit", ICounter.NOT_COVERED);
```

Comment:
@marchof excuse me that I provided wrong snippet - actually since base interface is not itialized in JDK 9 EA due to JI-9042842, it doesn't pass thru classloader and hence nor instrumented nor analyzed, resulting in `ICounter.EMPTY`. This time I checked locally, so hopefully this finally will be the last attempt :)